packages/cli/lib/developerTestAccounts.js

```diff
@@ -32,31 +32,31 @@ const getHasDevTestAccounts = appDeveloperAccountConfig => {
 
 const validateDevTestAccountUsageLimits = async accountConfig => {
   const accountId = getAccountId(accountConfig.portalId);
-  const response = await fetchDeveloperTestAccounts(accountId);
-  if (response) {
-    const limit = response.maxTestPortals;
-    const count = response.results.length;
-    if (count >= limit) {
-      const hasDevTestAccounts = getHasDevTestAccounts(accountConfig);
-      if (hasDevTestAccounts) {
-        throw new Error(
-          i18n('lib.developerTestAccount.create.failure.alreadyInConfig', {
-            accountName: accountConfig.name || accountId,
-            limit,
-          })
-        );
-      } else {
-        throw new Error(
-          i18n('lib.developerTestAccount.create.failure.limit', {
-            accountName: accountConfig.name || accountId,
-            limit,
-          })
-        );
-      }
+  const { data } = await fetchDeveloperTestAccounts(accountId);
+  if (!data) {
+    return null;
+  }
+  const limit = data.maxTestPortals;
+  const count = data.results.length;
+  if (count >= limit) {
+    const hasDevTestAccounts = getHasDevTestAccounts(accountConfig);
+    if (hasDevTestAccounts) {
+      throw new Error(
+        i18n('lib.developerTestAccount.create.failure.alreadyInConfig', {
+          accountName: accountConfig.name || accountId,
+          limit,
+        })
+      );
+    } else {
+      throw new Error(
+        i18n('lib.developerTestAccount.create.failure.limit', {
+          accountName: accountConfig.name || accountId,
+          limit,
+        })
+      );
```

Comment:
This isn't related, but I just flipped the if statement and returned `null` up front to reduce a layer of nesting